core/src/main/java/org/bitcoinj/params/MainNetParams.java

```diff
@@ -81,44 +81,694 @@ public MainNetParams() {
+                0x02,0x10,0x2a,0x03,0xb0,0xc0,0x00,0x03,0x00,0xd0,0x00,0x00,0x00,0x00,0x0f,0xd1,0x50,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x03,0xb0,0xc0,0x00,0x03,0x00,0xd0,0x00,0x00,0x00,0x00,0x0f,0xf1,0xd0,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x03,0xcf,0xc0,0x80,0x00,0x00,0x07,0x00,0x00,0x00,0x00,0x5f,0xd6,0x35,0x57,0x20,0x8d,
+                0x02,0x10,0x2a,0x04,0x21,0x80,0xdc,0x05,0x00,0x02,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x3b,0x20,0x8d,
+                0x02,0x10,0x2a,0x04,0x52,0xc0,0x01,0x03,0xc4,0x55,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8e,
+                0x02,0x10,0x2a,0x04,0xbc,0x40,0x1d,0xc3,0x00,0x8d,0x00,0x00,0x00,0x00,0x00,0x02,0x10,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x05,0x6d,0x40,0xb9,0x4e,0xd1,0x00,0xee,0xf4,0xbb,0xff,0xfe,0xce,0xa2,0xac,0x20,0x8d,
+                0x02,0x10,0x2a,0x05,0xd0,0x18,0x0a,0x75,0x6c,0x03,0x07,0x5b,0x2c,0x73,0x8c,0xaa,0x41,0x4b,0x20,0x8d,
+                0x02,0x10,0x2a,0x07,0x72,0x00,0xff,0xff,0x00,0x00,0x60,0xd1,0x0e,0xff,0xfe,0x09,0x38,0x86,0x20,0x8d,
+                0x02,0x10,0x2a,0x07,0xab,0xc4,0x00,0x00,0x00,0x00,0x00,0x89,0x02,0x34,0x01,0x80,0x01,0x94,0x20,0x8d,
+                0x02,0x10,0x2a,0x09,0x26,0x81,0x10,0x10,0x00,0x10,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x05,0x20,0x8d,
+                0x02,0x10,0x2a,0x09,0x26,0x81,0x01,0x02,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x02,0x10,0x20,0x8d,
+                0x02,0x10,0x2a,0x0b,0x48,0x80,0x00,0x01,0x00,0x00,0x02,0x25,0x90,0xff,0xfe,0xea,0x1c,0x20,0x20,0x8d,
+                0x02,0x10,0x2a,0x0e,0xb7,0x80,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x55,0xd1,0xf0,0x5b,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x0a,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x0b,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x1b,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x1c,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x1f,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x22,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x24,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x25,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x26,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2a,0x12,0x8e,0x40,0x56,0x68,0xe4,0x27,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x01,0x20,0x8d,
+                0x02,0x10,0x2c,0x0f,0xf8,0xf0,0xda,0x51,0x00,0x00,0x3a,0x45,0xfc,0x57,0x5e,0x30,0x25,0x93,0x20,0x8d,
```

Comment:
I know this is the same way that the Bitcoin Core C++ works, but I would suggest considering doing one or both of the following:
1. Moving the data to a Java resource file that is loaded via the classpath (in an Android-compatible way)
2. Using the original text-string format used in the `nodes_main.txt` file that is the input to the Python script.